packages/node_modules/@webex/internal-plugin-encryption/src/kms.js

```diff
@@ -250,6 +251,147 @@ const KMS = WebexPlugin.extend({
       });
   },
 
+  /**
+  * @typedef {Object} FetchPublicKeyResponse
+  * @property {number} status
+  * @property {UUID} requestId
+  * @property {string} publicKey
+  */
+  /**
+   * get public key from kms
+   * @param {Object} options
+   * @param {UUID} options.assignedOrgId the orgId
+   * @returns {Promise.<FetchPublicKeyResponse>} response of get public key api
+   */
+  fetchPublicKey({assignedOrgId}) {
+    this.logger.info('kms: fetch public key for byok');
+    console.warn(`---start to call fetchPublicKey in src ${assignedOrgId}`);
```

Comment:
remove console statements. @arun3528 @InteractiveTimmy we probably want to add prettier/lint rules for this...